### DISCUSSION TITLE
Is my student account package still active?

### DISCUSSION THREAD

**Discussion by voicijon13** [community/community#47301] [GitHub Education]
I was having trouble logging in with 2FV and unlink email. I am worried it ended my student account with GitHub Education Campus and hoping I can reactivate it as I am still a student wishing to utilize the additional resources and tools it offers.

**Comment by samus-aran** [comment]
Hi @voicijon13 ! The best place to go for answers to authenticate specific questions like this one is opening a ticket on our [Support page](https://support.github.com/request/account?tags=hubberfy_authentication). Please open a ticket with them :)
